server/modules/taxonomy/repository/taxonomyRepository.js

```diff
@@ -240,26 +240,27 @@ const deleteDraftTaxaByTaxonomyUuid = async (surveyId, taxonomyUuid, client = db
     [taxonomyUuid]
   )
 
-/**
- * If draft, returns something like prop_draft->>'propName', otherwise prop->>'propName'
- */
-const getPropCol = (propName, draft, columnPrefix = '') =>
-  `${columnPrefix} ${draft ? 'props_draft' : 'props'}->>'${propName}'`
-
 /**
  * Combines a draft and a published column prop, if needed, using the COALESCE function
  */
-const getPropDraftOrNot = (propName, draft, columnPrefix) =>
-  draft
+const getPropColCombined = (propName, draft, columnPrefix) => {
```

Comment:
nice. should this go to dbUtils? or not?

Reply:
right, I'll move it